### PR TITLE
fix: ensure contracts are only initialized once (C4 M-149)

### PR DESCRIPTION
**Pull request by pcarranzav**
Resolves https://github.com/code-423n4/2022-10-thegraph-findings/issues/149

### CODE REVIEW

**Review by tmigone — APPROVED** [review]

File: contracts/l2/token/L2GraphToken.sol [hunk 1]
```diff
@@ -45,7 +45,7 @@ contract L2GraphToken is GraphTokenUpgradeable, IArbToken {
      * - l1Address using setL1Address
      * @param _owner Governance address that owns this contract
      */
-    function initialize(address _owner) external onlyImpl {
+    function initialize(address _owner) external onlyImpl initializer {
```
[the remark this answers is not in the document]
Comment (pcarranzav) [reply contracts/l2/token/L2GraphToken.sol:48]: yes, `ERC20BurnableUpgradeable is Initializable`

[the remark this answers is not in the document]
Comment (tmigone) [reply contracts/l2/token/L2GraphToken.sol:48]: I find it hard to follow things like this when reading the code. It's a lot of steps you need to do to find out where an import is coming through (not that you are at fault lol).

**Review by abarmat — APPROVED** [review]
